actions/destinations/create.go

```diff
@@ -11,8 +11,17 @@ import (
 )
 
 // create will make a new destination
+// Count Destinations godoc
+// @Summary      Create a new destination
+// @Description  Create a new destination
+// @Tags		 Destinations
+// @Produce      json
+// @Param type query string false "type"
+// @Param reference_id query string false "reference_id"
```

Comment:
It can contain metadata.
"metadata := params.GetJSON(bux.ModelMetadata.String())"